s3.tf

```diff
@@ -46,6 +46,13 @@ resource "aws_s3_bucket" "apiary_data_bucket" {
       days          = lookup(var.apiary_managed_schemas[count.index], "s3_lifecycle_policy_transition_period", var.s3_lifecycle_policy_transition_period)
       storage_class = lookup(var.apiary_managed_schemas[count.index], "s3_storage_class", var.s3_storage_class)
     }
+
+    dynamic "expiration" {
+    for_each = lookup(var.apiary_managed_schemas[count.index], "s3_object_expiration_days", var.s3_object_expiration_days) != null ? [1] : []
+    content {
+      days = lookup(var.apiary_managed_schemas[count.index], "s3_object_expiration_days", var.s3_object_expiration_days)
```

Comment:
See my previous comments where I don't think we should have an overall default value for this.